### PR TITLE
Add options to opencensus bridge, and install tracer instead of returning

### PR DESCRIPTION
**Pull request by dashpole**
Part of https://github.com/open-telemetry/opentelemetry-go/issues/4562

Changes:

* Add options to trace and metric bridges
* Add trace option to specify TracerProvider, which defaults to the global TracerProvider.
* Add InstallTraceBridge(), which simplifies the user experience by setting the DefaultTracer in OpenCensus.
* Deprecate NewTracer

### CODE REVIEW

File: bridge/opencensus/config.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+// limitations under the License.
+
+package opencensus // import "go.opentelemetry.io/otel/bridge/opencensus"
+
+import (
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/trace"
+)
+
+const scopeName = "go.opentelemetry.io/otel/bridge/opencensus"
```
Comment (MadVikingGod) [inline bridge/opencensus/config.go:22]: Nit: This is defined here, but only used in `bridge.go`.  It should live there.

File: bridge/opencensus/bridge.go [hunk 2]
```diff
@@ -26,10 +26,21 @@ import (
 // NewTracer returns an implementation of the OpenCensus Tracer interface which
 // uses OpenTelemetry APIs.  Using this implementation of Tracer "upgrades"
 // libraries that use OpenCensus to OpenTelemetry to facilitate a migration.
+//
+// Deprecated: Use InstallTraceBridge instead.
 func NewTracer(tracer trace.Tracer) octrace.Tracer {
 	return internal.NewTracer(tracer)
 }
```
Comment (MadVikingGod) [inline bridge/opencensus/bridge.go:33]: Will this be removed before going stable?

**Review by MadVikingGod — APPROVED** [review]

File: bridge/opencensus/config.go (hunk 1 above)
Comment (dashpole) [reply bridge/opencensus/config.go:22]: It is used in metric.go as well (it was moved from there).

File: bridge/opencensus/trace.go [hunk 3]
```diff
@@ -26,10 +26,21 @@ import (
...
 	return internal.NewTracer(tracer)
 }
 
+// InstallTraceBridge installs the OpenCensus trace bridge, which overwrites
+// the global OpenCensus tracer implementation. Once the bridge is installed,
+// spans recorded using OpenCensus are redirected to the OpenTelemetry SDK.
+func InstallTraceBridge(opts ...TraceOption) {
+	cfg := newTraceConfig(opts)
+	tracer := cfg.tp.Tracer(scopeName)
+	octrace.DefaultTracer = internal.NewTracer(tracer)
```
Comment (MrAlias) [inline bridge/opencensus/trace.go:41]: Is it never the case that OC users want to use the tracer directly?

Comment (dashpole) [reply bridge/opencensus/trace.go:41]: The opencensus Tracer is an interface I added later to allow this bridge solution to work: https://github.com/census-instrumentation/opencensus-go/pull/1234.  I don't expect anyone actually ever uses it directly

Comment (MrAlias) [reply bridge/opencensus/trace.go:41]: Gotcha :+1:

**Review by MrAlias — APPROVED** [review]

**Pull request merged**